File: Assets/AltUnityTester/AltUnityDriver/Commands/InputActions/Tilt.cs
using Assets.AltUnityTester.AltUnityDriver.UnityStruct;

public class Tilt : AltBaseCommand
{
    Vector3 acceleration;
    public Tilt(SocketSettings socketSettings, Vector3 acceleration) : base(socketSettings)
    {
        this.acceleration = acceleration;
    }
    public void Execute()
    {
        string accelerationString = Newtonsoft.Json.JsonConvert.SerializeObject(acceleration, Newtonsoft.Json.Formatting.Indented, new Newtonsoft.Json.JsonSerializerSettings
        {
            ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore
        });
        Socket.Client.Send(toBytes(CreateCommand("tilt", accelerationString)));
        string data = Recvall();
        if (data.Equals("OK")) return;
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/InputActions/tilt.py
from altunityrunner.commands.base_command import BaseCommand
import time
class Tilt(BaseCommand):
    def __init__(self, socket,request_separator,request_end,x, y, z):
        super().__init__(socket,request_separator,request_end)
        self.x=x
        self.y=y
        self.z=z
    
    def execute(self):
        acceleration = self.vector_to_json_string(x, y, z)
        print ('Tilt with acceleration: ' + acceleration)
        data = self.send_data(self.create_command('tilt', acceleration ))
        return self.handle_errors(data)